client/scss/components/_image_gallery.scss

```diff
@@ -120,38 +123,44 @@
   .captioned-image__image-container {
     display: flex;
     flex-direction: column;
-    justify-content: center;
-    background: color('smoke');
+    justify-content: flex-end;
+    background: color('cream');
+  }
+
+  .captioned-image__caption {
+    position: relative;
+    opacity: 0;
+    z-index: 0;
```

Comment:
Yes – without it, a transparent, but higher-up-in-the-source-order element could prevent the correct '+more' control from being clicked.